server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -87,6 +87,70 @@ const virtualCardMutations = {
       return virtualCard;
     },
   },
+  createCardForCollective: {
+    description: 'Create new Stripe Virtual Card for existing hosted collective',
+    type: new GraphQLNonNull(VirtualCard),
+    args: {
+      name: {
+        type: GraphQLString,
+        description: 'Virtual Card name',
+      },
+      monthlyLimit: {
+        type: GraphQLInt,
+        description: 'Virtual Card monthly limit',
```

Comment:
Should we have a default here? And also maybe check the value make sense compared to what we're expecting? Maybe we should consider some maximum.

Reply:
Maybe, for now it's just a required attribute. Let me know what rules do you want.

Reply:
Maybe a constant like $1000 as a maximum for now.